python/prescription_generator.py

```diff
@@ -0,0 +1,699 @@
+import os
```

Comment:
this is c/c from pysin ?

Reply:
Yes exactly ! I found an old pysin package updated on pyp, but it doesn't contain the "prescription_generator" so I copy pasted it